lib/node_modules/@stdlib/math/base/special/bessely1/lib/index.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+/**
+* Compute the Bessel function of the second kind of order zero.
+*
+* @module @stdlib/math/base/special/bessely1
+*
+* @example
+* var PINF = require( '@stdlib/math/constants/float64-pinf' );
+* var NINF = require( '@stdlib/math/constants/float64-ninf' );
+* var y1 = require( '@stdlib/math/base/special/bessely1' );
+*
+* var v = y1( 0.0 );
+* // returns -Infinity
```

Comment:
This value does not match the REPL text.

Reply:
This one is correct. The other has been changed.